[cgroups]: bump lane versions

iholder101 opened the pull request:
Cgroups lanes only support k8s v1.23.
This PR creates lanes for 1.24-1.25 too.

brianmcarey reviewed (commented):
Looks good - would it be worth adding a periodic for 1.26 too? as we have periodics for 1.26 for other lanes in place already.

iholder101 commented:
> Looks good - would it be worth adding a periodic for 1.26 too? as we have periodics for 1.26 for other lanes in place already.
> 
> For example - k8s-1.26-sig-compute : https://testgrid.k8s.io/kubevirt-periodics#periodic-kubevirt-e2e-k8s-1.26-sig-compute

Done.
Also added presubmit lanes (which I've seem to forgot). I've added them for 1.26 too. WDYT?

brianmcarey commented:
> > Looks good - would it be worth adding a periodic for 1.26 too? as we have periodics for 1.26 for other lanes in place already.
> > For example - k8s-1.26-sig-compute : https://testgrid.k8s.io/kubevirt-periodics#periodic-kubevirt-e2e-k8s-1.26-sig-compute
> 
> Done. Also added presubmit lanes (which I've seem to forgot). I've added them for 1.26 too. WDYT?

The 1.26 presubmits have not been introduced yet due to an issue with istio and the 1.26 provider so I think the 1.26 cgroup presubmits should be removed from this PR.

iholder101 commented:
> > > Looks good - would it be worth adding a periodic for 1.26 too? as we have periodics for 1.26 for other lanes in place already.
> > > For example - k8s-1.26-sig-compute : https://testgrid.k8s.io/kubevirt-periodics#periodic-kubevirt-e2e-k8s-1.26-sig-compute
> > 
> > 
> > Done. Also added presubmit lanes (which I've seem to forgot). I've added them for 1.26 too. WDYT?
> 
> The 1.26 presubmits have not been introduced yet due to an issue with istio and the 1.26 provider so I think the 1.26 cgroup presubmits should be removed from this PR.

Makes sense :)
done.

brianmcarey reviewed (approved):
/approve

brianmcarey commented:
/test pull-project-infra-prow-deploy-test
